libqtile/config.py

```diff
@@ -568,6 +569,32 @@ def __repr__(self):
         return '<Match %s>' % self._rules
 
 
+class AndMatch(object):
+    """Combines multiple Match-Objects with logical &&-operation
+
+    Parameters
+    ==========
+    *matches:
+        Match-objects for combining
+    """
+    def __init__(self, *matches):
+        self.matches = []
+        self.matches.extend(matches)
+
+    def compare(self, client):
+        _tmp_bool = map(lambda x: x.compare(client), self.matches)
+        return reduce(lambda x, y: x and y, _tmp_bool )
```

Comment:
I feel like it may be better to use the `all()` builtin here, rather than `and`ing all of the results together